src/modules/Dropdown/Dropdown.js

```diff
@@ -650,7 +650,20 @@ export default class Dropdown extends Component {
 
     if (!search) return this.toggle(e)
     if (open) return
-    if (searchQuery.length >= minCharacters || minCharacters === 1) this.open(e)
+    if (searchQuery.length >= minCharacters || minCharacters === 1) {
+      this.open(e)
+      return
+    }
+    if (this.searchRef) this.searchRef.focus()
```

Comment:
@levithomason this method does not make me happy, do you have ideas how we can refactor it?